src/Sulu/Bundle/CategoryBundle/Domain/Event/CategoryLocaleAddedEvent.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Domain\Event;
+
+use Sulu\Bundle\CategoryBundle\Admin\CategoryAdmin;
+use Sulu\Bundle\CategoryBundle\Entity\CategoryInterface;
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+
+class CategoryLocaleAddedEvent extends DomainEvent
+{
+    /**
+     * @var CategoryInterface
+     */
+    private $category;
+
+    /**
+     * @var string
+     */
+    private $locale;
+
+    /**
+     * @var mixed[]
+     */
+    private $payload;
+
+    /**
+     * @param mixed[] $payload
+     */
+    public function __construct(
+        CategoryInterface $category,
+        string $locale,
+        array $payload
+    ) {
+        parent::__construct();
+
+        $this->category = $category;
+        $this->locale = $locale;
+        $this->payload = $payload;
+    }
+
+    public function getCategory(): CategoryInterface
+    {
+        return $this->category;
+    }
+
+    public function getEventType(): string
+    {
+        return 'locale_added';
```

Comment:
@sulu/core-team should we call this `locale_added` or `translation_added`? the type will also be used for medias, articles and pages

Reply:
i guess we should also use the same term for the `locale_copied` and `locale_removed` event

Reply:
i feel like `translation` makes more sense in the context of categories and from the perspective of the user. on the other hand, the `DocumentManager` already has a `copyLocale` and `removeLocale` method. so i guess both terms are okay for me 😅

Reply:
Since internally the term `locale` is used a lot more than `translation`, I would stick with that. On the frontend, the messages are translated anyway, so we can still use `Translation copied` and `Übersetzung kopiert` e.g.